examples/static_map/custom_type.cu

```diff
@@ -21,36 +21,38 @@
 #include <cuco/static_map.cuh>
 
 // User-defined key type
+#ifdef BUILD_PASCAL_CODE
```

Comment:
Can't we just check `CUDA_ARCH`?

Reply:
`__CUDA_ARCH__` can't work for the host side code.